requirements.txt

```diff
@@ -0,0 +1,3 @@
+gopenstates==6.11.1
```

Comment:
If possible, the OpenStates standard is to use a poetry project, not a straight `requirements.txt`. Can we do that instead?

Reply:
Also I'm pretty sure `gopenstates` is a typo.

Reply:
I've moved it to poetry now